subcommands/keys/tuf_updates_init.go

```diff
@@ -0,0 +1,58 @@
+package keys
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	initCmd := &cobra.Command{
+		Use:   "init",
+		Short: "Start a new transaction to update TUF root keys",
+		Run:   doTufUpdatesInit,
+	}
+	initCmd.Flags().StringP("changelog", "m", "", "Reason for doing this operation. Saved in root metadata to track change history.")
+	_ = initCmd.MarkFlagRequired("changelog")
+	initCmd.Flags().BoolP("first-time", "", false, "Used for the first customer rotation. The command will download the initial root key.")
+	initCmd.Flags().StringP("keys", "k", "", "Path to <offline-creds.tgz> used to store initial root key.")
```

Comment:
If a user keeps root and targets keys in separate archives then maybe we need to specify both of them in this command, or the targets keys are not needed?

Reply:
Only the first root key is saved here. There is no such thing as _the first offline targets key_.